protocols/forefront/index.json

```diff
@@ -0,0 +1,12 @@
+{
+  "cname": "forefront",
+  "description": "Forefront is an insight aggregator across the social token landscape.",
+  "path": "forefront",
+  "type": "snapshot",
+  "isEnabled": true,
+  "discourseForum": {
+    "url": "https://forum.forefront.news/",
```

Comment:
not sure about this bit, and hence not sure about the related id.

Reply:
no worries, remove this as of now. Not all protocols have discourse

Reply:
Yeah not sure that their forum uses Discourse but there's a slight chance. I would confirm with the team